mbed_tools_ci_scripts/tag_and_release.py

```diff
@@ -49,10 +53,11 @@ def tag_and_release(mode: CommitType, current_branch: Optional[str] = None) -> N
     if mode == CommitType.DEVELOPMENT:
         return
     _update_documentation()
+    licensing = _update_licensing_summary()
```

Comment:
Maybe a small comment here to say this has to go after updating documentation?

Reply:
aee6f7ee5b8763b2bacc9e5ff6f6923158220e52